README.md

```diff
@@ -74,30 +74,164 @@ An example of one such record:
 }
 ```
 
+
 ## Making Queries
 
+
 All queries to the index are made through HTTP using JSON data payloads.
 This gives a simple means of interaction that is easily accessible to any
 number of languages.
 
 These queries are handled via requests and wrapped into the index client.
 
-### Create a record
 
-***TODO***
+  ### Create a record ###
+
+
+# Method: create
+
+Required Arguments:
+  hashes (dict): {hash type: hash value,}
+      eg: ``hashes={'md5': ab167e49d25b488939b1ede42752458b'}``
+  size (int): file size metadata associated with a given uuid
+
+Optional Arguments:
+  did (str): GUID (Global Unique Identifier) for the new indexd to be made
+  urls (list): list of URLs where you can download the UUID
+  acl (list): access control list
+  authz (str): RBAC string
+  file_name (str): name of the file associated with a given UUID
+  metadata (dict): additional key value metadata for this entry
+  urls_metadata (dict): metadata attached to each url
+  baseid (str): optional baseid to group with previous entries versions
+  version (str): entry version string
+
+Return Value:
+  Document: indexclient representation of an entry in indexd
+
+Example: 
+
+ indexclient.create(
+    hashes = {'md5': ab167e49d25b488939b1ede42752458b'},
+    size = 5,
+    # optional arguments
+    acl = ["a", "b"]
+  )
+
+  ### Retrieve a record ###
+
+
+# Method: get
+
+Required Argument:
+  did (str): GUID (Global Unique Identifier) for the new indexd to be made
+
+Return Value:
+  Document: indexclient representation of an entry in indexd
```

Comment:
like I said above I would not include the required or return blocks. I would use actual values for the `did` arg though. like show `indexclient.get(did="dg.1234/03eed607-acb0-4532-b0ee-9e3766b1aa6e")